### DISCUSSION TITLE
Use as part of DEPNotify Routine?

### DISCUSSION THREAD

**Discussion by telepathine-ltd** [grahampugh/erase-install#103] [General]
Hi guys, I'm doing some testing now to see if I can fit this into our workflow for first boot - it works like a charm outside.

how might I integrate this into a first boot routine using DEPNotify? Would I put a command into a Jamf policy, and then call the custom trigger during DEPNotify? Would it then switch over to the erase-install.sh DEPNotify screen? Would the regular DEPNotify routine finish afterward? 

Inquiring minds want to know. Thanks again!

**Comment by OhadOliver** [comment]
+1

**Reply by grahampugh** [reply]
You'll have to test this out. For sure you can call erase-install from a Policy trigger. 

I designed the script with DEPNotify as a standalone display window, so I haven't played with how it would coincide/interfere with another DEPNotify workflow.

**Comment by MikePattie** [comment]
> Hi guys, I'm doing some testing now to see if I can fit this into our workflow for first boot - it works like a charm outside.
> 
> how might I integrate this into a first boot routine using DEPNotify? Would I put a command into a Jamf policy, and then call the custom trigger during DEPNotify? Would it then switch over to the erase-install.sh DEPNotify screen? Would the regular DEPNotify routine finish afterward?
> 
> Inquiring minds want to know. Thanks again!

Hey telepathine!

Did you end up playing around with this?
